qemu/tests/balloon_stress.py

```diff
@@ -18,12 +20,59 @@ def run(test, params, env):
     4) check device using right driver in guest.
     5) play online video in guest
     6) balloon memory in monitor in loop
-    7) check vm alive
+    7) kill video process and run it again (option)
+    8) check vm alive
 
     :param test: QEMU test object
     :param params: Dictionary with the test parameters
     :param env: Dictionary with test environment.
     """
+    def check_video_status(vm):
+        """
+        Check video playing status
+
+        :param vm: VM object
+        :return: return video player process
+        """
+        logging.info("Check if video is playing")
+        target_process = params.get("target_process", "wmplayer.exe")
+        check_cmd = params.get("check_process_cmd", "wmic process get name")
+        s, output = utils_misc.get_guest_cmd_status_output(vm, check_cmd)
+        process = re.findall(target_process, output, re.M | re.I)
+        if not process:
+            return None
+        else:
+            logging.info("Video process: %s" % process)
+            return process[0]
+
+    def run_video(vm):
```

Comment:
There is already a sub process when call run_video(), why create a process here again?